helm-charts/common/tgi/README.md

```diff
@@ -25,10 +25,10 @@ MODELNAME="/data/models--bigscience--bloom-560m"
 
 ## Values
 
-| Key           | Type   | Default                                           | Description                                                                                                                              |
-| ------------- | ------ | ------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| LLM_MODEL_ID  | string | `"bigscience/bloom-560m"`                         | Models id from https://huggingface.co/, or predownloaded model directory                                                                 |
-| port          | string | `"80"`                                            | Hugging Face Text Generation Inference service port                                                                                      |
-| global.volume | string | `"/mnt"`                                          | Cached models directory, tgi will not download if the model is cached here. The "volume" will be mounted to container as /data directory |
-| image         | string | `"ghcr.io/huggingface/text-generation-inference"` |                                                                                                                                          |
-| tag           | string | `"1.4"`                                           |                                                                                                                                          |
+| Key                     | Type   | Default                                           | Description                                                                                                                                                  |
+| ----------------------- | ------ | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| LLM_MODEL_ID            | string | `"bigscience/bloom-560m"`                         | Models id from https://huggingface.co/, or predownloaded model directory                                                                                     |
+| port                    | string | `2080`                                            | Hugging Face Text Generation Inference service port                                                                                                          |
```

Comment:
is this port number important? And jumped from 80 to 2080, a safer range?

Reply:
non-root users can not open port for listening under 1024, that's why we change it for safe.